src/tests/payments.js

```diff
@@ -0,0 +1,277 @@
+(function() {
+
+var assert = require('assert');
+var eq_ = assert.eq_;
+var contains = assert.contains;
+var mock = assert.mock;
```

Comment:
Might want to rename this var as `mock` doesn't really tell us much about what it does - hence wanting to know where the docs were.

Maybe mockModule or something?